source/cookbook/user_interface_and_interaction/displaying_formatted_dates_with_moment_js.md

```diff
@@ -29,10 +37,21 @@ Today's date: {{currentDate}}  // Today's date: August 30 2013
 
 You can even enhance your code and pass in the date format to the helper:
 
-```javascript
+```app/helpers/current-date.js
+export default Ember.Handlebars.makeBoundHelper(function() {
+  return moment().format('LL');
+});
+```
+
+```app/route/application.js
+import currentDate from 'app/helpers/current-date.js';
+
 Ember.Handlebars.registerBoundHelper('currentDate', function(format) {
```

Comment:
Please review this section (about to push now). I set up the "makeBoundHelpers" and "registerBoundHelper" based on what was working for me in the jsbin (the ember-cli prototype).

I am assuming this means "remove the import" but not sure what exact line number this is pointing to (or the whole section?)